code/game/atoms.dm

```diff
@@ -132,6 +132,9 @@
 /atom/proc/is_open_container()
 	return flags & OPENCONTAINER
 
+atom/proc/is_transparent()
+	return flags & TRANSPARENT
+

```

Comment:
No, that would not be helpful. These things are better as procs because there might be conditions in which the atoms lose the TRANSPARENT flag and it's better to program those conditions in the is_transparent() proc.